kbase-extension/static/kbase/templates/data_staging/ftp_file_table.html

```diff
@@ -1,41 +1,46 @@
-<table id="kb-data-staging-table" class="table table-striped table-condensed" style="table-layout:fixed">
-    <thead>
+<table class="kb-staging-table table table-striped table-condensed">
+    <thead class="kb-staging-table-header">
         <tr>
-            <th style="width: 5%"></th>
-            <th style="width: 40%; overflow:hidden">Name</th>
-            <th style="width: 13%">Size</th>
-            <th style="width: 12%">Age</th>
-            <th style="width: 30%">Import As...</th>
+            <!-- TODO: what behavior should select all checkbox have? -->
+            <th class="kb-staging-table-header__checkbox">
+                <input type="checkbox" name="select_all" value="1" id="">
+            </th>
+            <th class="kb-staging-table-header__file"></th>
```

Comment:
do these need content for any poor person who is trying to use the UI without the benefit of full 20/20 vision? (I don't know what the policy is on table headers, so "no" is a perfectly acceptable answer)

Reply:
Here is some info about table accessibility: https://developer.mozilla.org/en-US/docs/Learn/HTML/Tables/Advanced

Reply:
i compromised with an aria-label here since we don't want to display the table header text.